java/rocksjni/portal.h

```diff
@@ -6396,6 +6444,35 @@ class TxnDBWritePolicyJni {
   }
 };
 
+// The portal class for org.rocksdb.ValueType
+class EntryTypeJni {
+ public:
+  static jbyte toJavaEntryType(const ROCKSDB_NAMESPACE::EntryType& entry_type) {
+    switch (entry_type) {
+      case ROCKSDB_NAMESPACE::EntryType::kEntryPut:
+        return 0x0;
+      case ROCKSDB_NAMESPACE::EntryType::kEntryDelete:
+        return 0x1;
+      case ROCKSDB_NAMESPACE::EntryType::kEntrySingleDelete:
+        return 0x2;
+      case ROCKSDB_NAMESPACE::EntryType::kEntryMerge:
+        return 0x3;
+      case ROCKSDB_NAMESPACE::EntryType::kEntryRangeDeletion:
+        return 0x4;
+      case ROCKSDB_NAMESPACE::EntryType::kEntryBlobIndex:
+        return 0x5;
+      case ROCKSDB_NAMESPACE::EntryType::kEntryDeleteWithTimestamp:
+        return 0x6;
+      case ROCKSDB_NAMESPACE::EntryType::kEntryWideColumnEntity:
+        return 0x7;
+      case ROCKSDB_NAMESPACE::EntryType::kEntryOther:
+        return 0x8;
+      default:
+        return 0x9;
```

Comment:
There does not appear to be a `0x9` defined in the Java enum, so this will raise an exception. Is that intentional?

Reply:
yeah